src/core/ext/filters/client_channel/resolver/dns/c_ares/grpc_ares_ev_driver.cc

```diff
@@ -90,15 +90,18 @@ static void grpc_ares_notify_on_event_locked(grpc_ares_ev_driver* ev_driver);
 
 static grpc_ares_ev_driver* grpc_ares_ev_driver_ref(
     grpc_ares_ev_driver* ev_driver) {
-  gpr_log(GPR_DEBUG, "Ref ev_driver %" PRIuPTR, (uintptr_t)ev_driver);
+  GRPC_CARES_TRACE_LOG("request:%p Ref ev_driver %" PRIuPTR, ev_driver->request,
```

Comment:
Why not use `%p` for `ev_driver`?

Same thing throughout this file.

Reply:
Done. Yeah I think it should have been %p before.